src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
Good point, yes should have cleared the vector too..

Keeping a vector of `ScopedConnection`s was problematic last time I tried it, but I can try it again as it seems like there is no reason it shouldn't work if we're `std::move()`ing things into it.